controllers/MessageController.ts

```diff
@@ -41,13 +41,19 @@ export default class MessageController implements MessageControllerI {
                 .findAllMessagesReceivedByUser);
 
             app.get("/api/users/:uid/messages", MessageController.messageController
-                .findAllMessagesSentByUser)
+                .findAllMessagesSentByUser);
+
+            app.get("/api/users/:gid/messages", MessageController.messageController
```

Comment:
This route is indistinguishable from the route above it since uid and gid are both mongo ids and everything in the URL is the same.
We can resolve this issue by changing "users" to "groups"